src/browser/screens/Check.react.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import Component from 'react-pure-render/component';
+
+import LeftItem from '../components/LeftItem.react';
+import JobDescription from '../components/JobDescription.react';
+import boundScroll from '../lib/boundScroll';
+import Chevron from '../components/Chevron.react';
+import data from '../data/check.json';
+
+if (process.env.IS_BROWSER) {
+  require('./Check.styl');
+}
+
+@boundScroll()
+export default class Check extends Component {
+  static propTypes = {
+    actions: React.PropTypes.object.isRequired,
+    candidate: React.PropTypes.object.isRequired
+  }
+
+  proceed() {
+    const {actions: {nextScreen}} = this.props;
+    nextScreen();
+  }
+
+  handleChange(feature) {
+    const {actions: {saveFeature}} = this.props;
+    saveFeature(feature);
+  }
+
+  render() {
+    const specificData = data[this.props.candidate.role.id] || [];
+    const backpackUrl = {
+      img: "../../../assets/img/checkifyouhaveit.svg"
+    };
+
+    return (
+      <section className="check-screen screen">
+        <div className="container">
+          <LeftItem data={backpackUrl} />
+          <div className="right">
+            <div className="content">
+              <div className="screen-content">
+                <h1>check if we look in the same direction</h1>
```

Comment:
Maybe `Let's see if we look in the same direction`? "Check" sounds a bit out of place